agent/consul/state/usage.go

```diff
@@ -99,11 +99,20 @@ func updateUsage(tx WriteTxn, changes Changes) error {
 
 		switch change.Table {
 		case tableNodes:
```

Comment:
Reviewers: is checking for `PeerName` field all that's needed to catch if these objects are imported?

Reply:
Yep, should be enough.